pkg/batchjob/stat.go

```diff
@@ -0,0 +1,55 @@
+package batchjob
+
+import (
+	"errors"
+	"log"
+	"time"
+
+	"github.com/dyweb/sundial/pkg/models"
+	"github.com/dyweb/sundial/pkg/store/rdb"
+	"github.com/dyweb/sundial/pkg/store/tsdb"
+)
+
+//StatWorker collects heartbeats from tsdb and writes stat to rdb
+//collects heartbeats from `startTime` with a given range length
+func StatWorker(rdbStore rdb.Store, tsStore tsdb.Store, beginTime time.Time, statRange models.StatRange) {
+	hbs, err := tsStore.QueryHeartBeats(beginTime, beginTime.Add(toDuration(statRange)))
+	if err != nil {
+		log.Fatalln(err)
```

Comment:
Why you are using log.Fataln, this will call `os.Exit(1)`. Wrap the error and return to caller should be a better idea, if you run this in your server, the server will stop when there is error for cron. I didn't see a separated main package for the cron so I assume you are running it in server.

Also I didn't see a stat_test.go, besides there is no unit test, I am curious how you are testing current code?